src/main/webapp/management-console.js

```diff
@@ -93,11 +93,10 @@ angular.module('managementConsole', [
                         cluster.nodeCount += host.count;
                       });
 
-                      if (serversInGroup.some(isStopped) && serversInGroup.some(isRunning)) {
-                        cluster.status = 'DEGRADED';
-                      }
-                      else if (serversInGroup.some(isRunning)) {
-                        cluster.status = 'STARTED';
```

Comment:
@tristantarrant and here for server group. If at least one server is running and all servers have the same coordinator then there is not split. In that case we mark server group 'STARTED'. Otherwise if coords are different we have a split. In that case we mark it as 'DEGRADED'. If no servers in the group are running then it is STOPPED. This needs to be tested - will do